pkg/daemon/config_drift_monitor.go

```diff
@@ -277,58 +278,60 @@ func (c *configDriftWatcher) handleFileEvent(event fsnotify.Event) error {
 // Validates on disk state for potential config drift.
 func (c *configDriftWatcher) checkMachineConfigForEvent(event fsnotify.Event) error {
 	// Ignore events for files not found in the MachineConfig.
-	if !c.filePaths.Has(event.Name) {
+	mcPath, ok := c.filePaths[event.Name]
+	if !ok {
 		return nil
 	}
 
-	if err := validateOnDiskState(c.MachineConfig, c.SystemdPath); err != nil {
```

Comment:
suggestion: An unintended side-effect of this change is that we no longer have test coverage for the `validateOnDiskState()` function. Since Config Drift Monitor is not the only code which calls that path, we should add a small smoke test to ensure that it continues to work correctly.

Reply:
`validateOnDiskState()` continues to be used here: https://github.com/openshift/machine-config-operator/blob/d59a67ffb38d62e329a11b62b8632a3e12666718/pkg/daemon/daemon.go#L2559

Reply:
working on adding the coverage for this function apart